src/components/Settings/ExtraOrders.tsx

```diff
@@ -0,0 +1,162 @@
+import Locale from "@/locales";
+import { useState, Dispatch } from "react";
+
+export default function ExtraOrders({ t }: { t: Locale }) {
+  const [drinkCoupons, setDrinkCoupons] = useState(20);
+  const [tables, setTables] = useState(20);
+  const [chairs, setChairs] = useState(20);
+  const [representatives, setRepresentatives] = useState(20);
+  const [sitting, setSitting] = useState(20);
+  const [editState, setEditState] = useState(false);
+
+  function plusMinus(num: number, set: Dispatch<number>) {
```

Comment:
Would be nice to have a `step` parameter here, since drink coupons will only be able to be bought in tens.

Reply:
I've added it now.